common/src/main/java/com/lightstep/tracer/shared/CollectorClient.java

```diff
@@ -0,0 +1,46 @@
+package com.lightstep.tracer.shared;
+
+import com.lightstep.tracer.grpc.CollectorServiceGrpc;
+import com.lightstep.tracer.grpc.CollectorServiceGrpc.CollectorServiceBlockingStub;
+import com.lightstep.tracer.grpc.ReportRequest;
+import com.lightstep.tracer.grpc.ReportResponse;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import java.util.concurrent.TimeUnit;
+
+public class CollectorClient {
```

Comment:
I have a TODO to investigate and properly implement this class later, I have a feeling the GRPC client will need to be more sophisticated (connection management?), warranting a separate class. For now I want to see a working GRPC connection to the collector first - if it really is *this* easy in the end I'll get rid of the class :)